src/universal/decorators/requireAuth/requireAuth.js

```diff
@@ -3,7 +3,7 @@ import {push} from 'react-router-redux';
 import {localStorageVars} from '../../utils/clientOptions';
 
 let key;
-export default ComposedComponent => {
```

Comment:
May be worthwhile causing some ruckus over in eslint land to let em know how silly the rule is: https://github.com/eslint/eslint/issues/4353